backend/file_management/views.py

```diff
@@ -44,7 +44,9 @@ class FileManagementViewSet(viewsets.ModelViewSet):
     """
 
     versioning_class = URLPathVersioning
-    queryset = ConnectorInstance.objects.all()
+
+    def get_queryset(self):
+        return ConnectorInstance.objects.all()
```

Comment:
@muhammad-ali-e NIT QQ : Any idea why this cached queryset was working in v1?

Reply:
It could be related to the schema switching that occurs with each call. I haven't investigated it thoroughly yet, but there may also be some corner cases involved.